docs/cmdline-opts/remove-on-error.d

```diff
@@ -12,4 +12,4 @@ When curl returns an error when told to save output in a local file, this
 option removes that saved file before exiting. This prevents curl from
 leaving a partial file in the case of an error during transfer.
 
-If the output is not a file, this option has no effect.
+If the output is not a regular file, this option cannot do its job.
```

Comment:
"cannot" makes it seems like it will try to do its job but won't succeed. I prefer the original working (but say "regular file") with this change.